tsconfig.base.json

```diff
@@ -16,14 +16,14 @@
     "types": [],
     "paths": {
       "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"],
```

Comment:
Probably super picky for such nice work

But shove this one to the end to keep it ordered A-Z